src/components/Disputes/panels/CommitPanel.js

```diff
@@ -54,12 +56,20 @@ const CommitPanel = React.memo(function CommitPanel({
   const handleDownloadCode = useCallback(() => {
     download()
     setCodeSaved(true)
-  }, [download])
+    localStorage.setItem(
+      `oneTimeCode-${connectedAccount}-${dispute.id}`,
+      oneTimeCode
+    )
+  }, [connectedAccount, dispute.id, download, oneTimeCode])
 
   const handleCopyCode = useCallback(() => {
     setCodeCopied(true)
     toast('One-time-use code copied')
-  }, [toast])
+    localStorage.setItem(
+      `oneTimeCode-${connectedAccount}-${dispute.id}`,
```

Comment:
Since we do this twice, perhaps it would make sense to move to the handler generating the tx?